### PR TITLE
Show Sendmail settings on admin config page

### PR DESCRIPTION
**Pull request by cez81**
As title add the Sendmail setting to the config page
sendmail

### CODE REVIEW

File: templates/admin/config.tmpl [hunk 1]
```diff
@@ -170,10 +170,19 @@
...
-					<dt>{{.i18n.Tr "admin.config.mailer_host"}}</dt>
-					<dd>{{.Mailer.Host}}</dd>
+					{{if not .Mailer.UseSendmail}}
+						<dt>{{.i18n.Tr "admin.config.mailer_disable_helo"}}</dt>
+						<dd><i class="fa fa{{if .Mailer.DisableHelo}}-check{{end}}-square-o"></i></dd>
+						<dt>{{.i18n.Tr "admin.config.mailer_host"}}</dt>
+						<dd>{{.Mailer.Host}}</dd>
+					{{else}}
+						<dt>{{.i18n.Tr "admin.config.mailer_use_sendmail"}}</dt>
+						<dd><i class="fa fa{{if .Mailer.UseSendmail}}-check{{end}}-square-o"></i></dd>
```
Comment (sapk) [inline templates/admin/config.tmpl]: The if .Mailer.UseSendmail at this line could be removed since we check it ar L173.

**Comment by sapk** [comment]
Just a `if` that could be removed. For the rest LGTM.

File: templates/admin/config.tmpl (hunk 1 above)
Comment (cez81) [reply templates/admin/config.tmpl]: Done

**Review by sapk — APPROVED** [review]

**Comment by lafriks** [comment]
LGTM

**Comment by lafriks** [comment]
Make LG-TM work

**Pull request merged**